Src/Plugins/Security/HelpersUtiliesAndExtensions/DomainFilteringHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.HelpersUtiliesAndExtensions
```

Comment:
>HelpersUtiliesAndExtensions [](start = 71, length = 27)

There is a misspelling in this namespace. Just use Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security,that's fine.

Reply:
Fixed the namespace name, but left a message on chat about whether to move the actual file.

---
In reply to: [575560622](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575560622) [](ancestors = 575560622)